src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -183,38 +169,41 @@ void postCommentPerIssue(PullRequestRef pr, String sonarQubeURL,
 
       // Let's call this "issue_loop"
       for (Issue issue : issues) {
-        StashCommentReport comments = commentsByFile.get(issue.getPath());
+        String path = getIssuePath(issue);
+        StashCommentReport comments = commentsByFile.get(path);
+        String commentContent = MarkdownPrinter.printIssueMarkdown(issue, config.getSonarQubeURL());
+        Integer issueLine = issue.line();
+        // FIXME move this somewhere else
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1134)